doc/source/ray-core/objects/object-spilling.rst

```diff
@@ -68,6 +68,28 @@ If you are using an HDD, it is recommended that you specify a large buffer size
         },
     )
 
+To prevent running out of disk space, local object spilling will throw ``OutOfDiskError`` if the disk utilization exceeds the predefined threshold.
+If multiple physical devices are used, any physical device's over-usage will trigger the ``OutOfDiskError``.
```

Comment:
Should we mention it is for /tmp mounted devices?

Reply:
hmm this is not true right; we monitor the disk where the spilling is configured